src/utils/displayResult.js

```diff
@@ -2,25 +2,39 @@ import chalk from 'chalk'
 
 export function displayResult(err, failureMessage, successMessage) {
   if (err) {
-    if (err.hasOwnProperty('body')) {
-      try {
-        const body = JSON.parse(err.body)
-        const message = body.message || ''
-        const details = body.details || ''
+    if (err.hasOwnProperty('error')) {
+      let body = err.error || null
 
-        console.log(
-          chalk.redBright(
-            failureMessage,
-            `${message}${details ? '\n' + details : ''}`
-          )
-        )
-      } catch (parseError) {
-        console.log(chalk.redBright('Unable to parse error\n', parseError))
-        console.log(chalk.redBright(failureMessage, err.body))
+      if (err.error) {
+        body = err.error
       }
+
+      if (!body) {
+        try {
+          body = JSON.parse(err.error)
```

Comment:
If `body` is falsy, doesn't that automatically mean that `err.error` is falsy since it's assigned to `body`?

Reply:
Good point, this whole function need revisiting